src/lib/libdylink.js

```diff
@@ -612,6 +612,7 @@ var LibraryDylink = {
     '$currentModuleWeakSymbols',
     '$updateTableMap',
     '$wasmTable',
+    '$__ATPOSTCTOR__',
```

Comment:
Where is this needed?

Reply:
Down below in some of the initialization code.

Reply:
Can we call `addOnPostCtor` instead?    I think it would be good if we use those helpers everywhere (or nowhere).